src/modules/ShortcutGuide/ShortcutGuide/main.cpp

```diff
@@ -54,7 +54,15 @@ int WINAPI wWinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance,
         });
     }
 
-    auto window = OverlayWindow();
+    auto hwnd = GetForegroundWindow();
+    auto window = OverlayWindow(hwnd);
+    if (window.IsDisabled())
```

Comment:
It would be better to run this check before creating `OverlayWindow`

Reply:
Moved windows creating to the `ShowWindow` method. We still need to initialize `OverlayWindow` to retrive a list of disabled apps.